cfgov/login/urls.py

```diff
@@ -35,7 +34,7 @@
     ]
 else:
     urlpatterns = [
-        re_path(r"^login/$", login_with_lockout, name="cfpb_login"),
+        re_path(r"^login/$", LoginView.as_view(), name="cfpb_login"),
```

Comment:
If we deprecate our custom view, could we remove this definition, along with the (likely unused) `^django-admin/login/$` pattern, and the logout pattern, since these are already defined in [the wagtail admin URLs](https://github.com/wagtail/wagtail/blob/v2.15.6/wagtail/admin/urls/__init__.py)?

Reply:
The Wagtail admin adds `/admin/login`, but we use `/login`, so I've left that URL in place, but now it uses Wagtail's `LoginView`.

Reply:
I'm loath to change user-facing paths here, with untestable implications for our SAML2 implementation. I could change it to a redirect to `/admin/login` when SAML2 is not enabled.